bin/build.js

```diff
@@ -3,21 +3,46 @@
 const fs = require('fs');
 
 let original = '';
+let version = '';
 
+// Read version from the normalize.css package and write it in our package.json
+const normalizePackage = fs.createReadStream('./node_modules/normalize.css/package.json', 'utf8');
+
+normalizePackage.on('data', (chunk) => {
+  if (chunk.indexOf("version: ")) {
+    const match = chunk.match(/("version": "[\d+\.]*")/g)[0];
+    version = match.match(/[\d\.*]+/g)[0];
+  }
+}).on('end', () => {
+  fs.readFile('./package.json', 'utf8', function (error, data) {
+    if (error) {
+      return console.log(error);
+    }
+    var result = data.replace(/("version": "[\d+\.]*")/g, `"version": "${version}"`);
+
+    fs.writeFile('./package.json', result, 'utf8', function (error) {
+       if (error) {
+        return console.log(error);
+      };
+    });
+  });
+});
+
+// Read css from normalize.css, clean comments and write it in our index.js
 const readStream = fs.createReadStream('./node_modules/normalize.css/normalize.css', 'utf8');
```

Comment:
Since you're already reading the package.json, why not also check the main property and use it here?

Reply:
Can you explain this a bit? Not sure I understand what you mean...

Reply:
When you require/import a dependency like `require('foo')`, node will do the following:
1. Check if `node_modules/foo/package.json` exists
2. If it does, check if it contains the `main` property
3. Load the dependency by using the path `node_modules/foo/<main>`

If any of the steps fail, it will default to `node_modules/foo/index.js`.
So by using the `main` property, you avoid the hardcoding of `'./node_modules/normalize.css/normalize.css'` which could change at any minor or patch release of normalize.css

Docs: https://docs.npmjs.com/files/package.json#main